go.mod

```diff
@@ -0,0 +1,68 @@
+module sigs.k8s.io/cluster-api-operator
```

Comment:
Can you run go mod tidy on this? I thought all the indirect things should be in one block below but it seems mixed right now

Reply:
done, it should look good now